test/e2e/kubernetes/pod/pod.go

```diff
@@ -235,9 +235,10 @@ func RunCommandMultipleTimes(podRunnerCmd podRunnerCmd, image, name, command str
 // GetAll will return all pods in a given namespace
 func GetAll(namespace string) (*List, error) {
 	cmd := exec.Command("kubectl", "get", "pods", "-n", namespace, "-o", "json")
-	util.PrintCommand(cmd)
```

Comment:
We don't need to output a bunch of `kubectl get pods` statements in our E2E stdout